deployment/src/test/java/io/quarkiverse/logging/splunk/LoggingSplunkMandatoryConfigTest.java

```diff
@@ -21,6 +21,7 @@ class LoggingSplunkMandatoryConfigTest {
             .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class))
             .withConfigurationResource("application-splunk-logging-mandatory.properties")
             .assertException(e -> {
+                System.out.println(e.getMessage());
```

Comment:
intended?

Reply:
hmm probably not no, was just the test was failing and it was not logging the exception so I could not see what was the issue